python/pyspark/sql/tests/test_functions.py

```diff
@@ -81,7 +81,9 @@ def test_function_parity(self):
         missing_in_py = jvm_fn_set.difference(py_fn_set)
 
         # Functions that we expect to be missing in python until they are added to pyspark
-        expected_missing_in_py = set()
+        expected_missing_in_py = set(
+            ["make_time"]  # TODO(SPARK-52888): @uros-db is addressing this separately.
```

Comment:
Let's don't assign tasks in the source code.
```suggestion
            # TODO(SPARK-52888): Implement the make_time function in Python
            ["make_time"]
```